dts-generator/src/main/java/com/telerik/dts/DtsApi.java

```diff
@@ -888,15 +889,43 @@ private String getMethodName(Method m) {
     }
 
     private String getMethodParamSignature(JavaClass clazz, TypeDefinition typeDefinition, Method m) {
+        Pattern pattern = Pattern.compile("\\s" + Pattern.quote(m.getName()) + "\\((.*)\\)\\s");
+        Matcher matcher = pattern.matcher(m.toString() + " ");
+        List<String> params = Arrays.asList();
+        if (matcher.find()) {
+            params = Arrays.asList(matcher.group(1).split("\\,\\s"));
+            if (matcher.find()) {
+                System.out.println("🆘 getMethodParamSignature: Second matcher.find() -> " + m.toString());
```

Comment:
Seems like you have a weird extra "SOS" Unicode character here at the front of the println.